README.md

```diff
@@ -16,14 +16,20 @@ This repository contains the necessary scripts and configuration files to start
     conda activate myenv
     ```
 
-3. Change permissions for the shell scripts to make them executable:
+3. Install the package. This will install dependencies.
+
+    ```bash
+    python -m pip install .
+    ```
+
+4. Change permissions for the shell scripts to make them executable:
```

Comment:
I think we can delete step 4 or make it an optional step with a broader command (e.g. `find . -type f -name "*.sh" -exec chmod +x {} \;`)